src/dra/5298C.c

```diff
@@ -1337,9 +1337,37 @@ INCLUDE_ASM("asm/us/dra/nonmatchings/5298C", func_800FA3C4);
 
 INCLUDE_ASM("asm/us/dra/nonmatchings/5298C", func_800FA60C);
 
-// DECOMP_ME_WIP func_800FA7E8 https://decomp.me/scratch/JL0hI
-// has some logic related to the weapon struct
-INCLUDE_ASM("asm/us/dra/nonmatchings/5298C", func_800FA7E8);
+// If you use both attack buttons at once, see if something special happens.
+// Applies to Shield Rod + Shield, or dual Heaven Swords
+void CheckWeaponCombo(void) {
+    s32 weapon0;
+    s32 weapon1;
+    s32 combo1;
+    s32 combo2;
+    s32 comboBits;
+    s32 i;
+    s32 oddComboCheck;
```

Comment:
What's the meaning of `odd` here? That the code for the check is strange?

Reply:
Yes, that's correct. Hopefully when we know the proper usage of D_8013AEE4 we can rename the variable to something that usefully describes the purpose of that.